db/migrate/20231026034304_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postCounter
+
```

Comment:
## [ Optional ] ♻️
I highly recommend you work on this change 

- [ ] For consistency in naming your fields, using the underscore naming convention for joined words would be nice. `post_counter` instead of the camel case style.